samples/Mvc.LocalizationSample.Web/Resources/Models.Product.fr.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="MaxLengthTwo" xml:space="preserve">
+    <value>{0} doivent avoir une longueur d'au plus deux.</value>
```

Comment:
This is a strange pattern to encourage. If user wants any language but French, they'll see "MaxLengthTwo" as the error message. Why aren't the localization keys useful to end users?